src/cow-react/modules/trade/hooks/setupTradeState/useSetupTradeState.ts

```diff
@@ -47,6 +48,10 @@ function shouldSkipUpdate(tradeStateFromUrl: TradeState, tradeStateFromStore: Tr
   return chainIdIsNotChanged && recipientIsNotChanged && inputCurrencyIsNotChanged && outputCurrencyIsNotChanged
 }
 
+function compareChainIds(aChainId: Nullish<number>, bChainId: Nullish<number>): boolean {
```

Comment:
I suggest you follow the same pattern as the fn above `areCurrenciesTheSame` and name it: `areChainIdsTheSame`

Reply:
Thank you!